src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -128,6 +128,7 @@ public RobotMap() {
 		Component.driverXbox.setDeadZone(DefaultDriver.XBOX_MINIMUM_THRESHOLD);
 		// Main Subsystems
 		Component.alignCamera = new AligningCamera(PIDSourceType.kRate);
+		Component.shifter = new AutoSolenoidShifters(Port.Pneumatics.solenoidUp, Port.Pneumatics.solenoidDown);
```

Comment:
Why is this initialization (`Component.shifter`) done twice? (On line 131 and line 99)